### DISCUSSION TITLE
Are reserved link ids unreserved after some time if they are not used?

### DISCUSSION THREAD

**Discussion by FreePhoenix888** [deep-foundation/Discussions#33] [Q&A] [answered]
Are reserved link ids unreserved after some time if they are not used?

**Comment by konard** [comment, marked as answer]
No, until https://github.com/deep-foundation/deeplinks/issues/194 is resolved.

At the moment we should minimize the usage of reserved links, to reduce the speed of reducing the number of available links addresses.
